content/shared/influxdb3-admin/tokens/admin/_index.md

```diff
@@ -1,6 +1,3 @@
-  
-Manage admin tokens in your {{< product-name >}} instance.
-An admin token grants
-access to all actions (CLI commands and API endpoints) for the server.
+<!-->  
```

Comment:
Do we need this comment?

Reply:
Without the comment, I get:

```
Error: error building site: process: readAndProcessContent: "/Users/ja/Documents/github/docs-v2/content/shared/influxdb3-admin/tokens/admin/_index.md:2:1": unmarshal failed: invalid character '{' looking for beginning of object key string
```

...even if I leave blank lines before the shortcode.